src/framework/bootstrap.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+// ---------------------------------------------------------
+// Note: this code would usually be provided by a framework.
+// ---------------------------------------------------------
+
+import {useState, Suspense, use} from 'react';
```

Comment:
Are the react imports necessary in this file (bootstrap.js)?

Reply:
removed https://github.com/reactjs/server-components-demo/commit/99e49cbb37a6e73aaa0d0751d7cee9217c263c26